addons/project/static/src/scss/project_widgets.scss

```diff
@@ -74,6 +74,27 @@
             }
         }
     }
+    @for $size from 2 through length($o-colors) {
+        // Note: the first color is supposed to be invisible so it's ignored
+        .oe_kanban_color_#{$size - 1} {
+            border-left-color: nth($o-colors, $size);
+            &:after {
+                background-color: nth($o-colors, $size);
+            }
+        }
+    }
```

Comment:
Already defined here : https://github.com/odoo/odoo/blob/a48a30f954afcb6ff3a59c4f32b05fd0c2cfcd2b/addons%2Fweb%2Fstatic%2Fsrc%2Flegacy%2Fscss%2Futils.scss#L274-L284

It seems useless to me to re-define it, isn't it ?

Reply:
Done.